osgar/lib/quaternion.py

```diff
@@ -47,4 +47,12 @@ def heading(quaternion):
     az =  math.atan2(2*(w0*z0+x0*y0), 1-2*(y0*y0+z0*z0))
     return az
 
+def from_axis_angle(axis, angle):
```

Comment:
I would add maybe some comment ...

Reply:
Well, I thought it is self explaining - make quaternion from axis and angle. Or did you have something else in mind?